gammapy/estimators/map/core.py

```diff
@@ -1138,6 +1146,16 @@ def slice_by_coord(self, slices):
         -------
         flux_maps : `FluxMaps`
             Sliced flux maps object.
+
+        Examples
+        --------
+        >>> from gammapy.estimators import FluxPoints
+        >>> import astropy.units as u
+        >>> lc_1d = FluxPoints.read(
+        ...     "$GAMMAPY_DATA/estimators/pks2155_hess_lc/pks2155_hess_lc.fits", format="lightcurve"
```

Comment:
Do you need to explicitly mention the format?

Reply:
I don't think so but it was specified in the tutorial it uses, so I thought best to be consistent?

Reply:
The tutorial was probably not updated

Reply:
I will adjust it so that there is no 'format' option